install.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# golang check for version specified in go.mod
+echo "[*] Checking golang"
+go version
+if [ $? -ne 0 ]; then
+    echo "Failed to check golang version: do you have go installed?"
+    exit 1
+fi
+echo "[OK] golang installed"
+
+# Building and installing sgrep
+echo "[*] Building and installing sgrep"
+go  build -o sgrep
+if [ $? -ne 0 ]; then
+    echo "Failed to build sgrep"
+    exit 1
+fi
+echo "[OK] sgrep built"
+
+# Moving sgrep to /usr/bin/sgrep
+sudo cp sgrep /usr/bin/sgrep
```

Comment:
Not sure if some people would want this. Also, asking for the user's password without an explanation why is kind of a red flag in my opinion.

Maybe we can give this as an option flag?

Reply:
Makes sense, I am on it. Do you think it would be ok to have three options:
- Move to /usr/bin/sgrep
- Add current path to $PATH
- Leave everything as it is

With a sudo -v at the beginning stating the reason?

Reply:
Yeah I like that. For me, personally, I just want to use it within the repo directory. I don't want system wide or user wide usage.